repos.json

```diff
@@ -38,21 +39,21 @@
     "protoparser": ["Java"],
     "rack-servlet": ["Java"],
     "rails-auth": ["Ruby"],
-    "retrofit": ["Android", "Java"],
+    "retrofit": ["Android", "Java", "Kotlin"],
     "seismic": ["Android"],
     "shift": ["Other"],
     "shuttle": ["Other"],
     "spacecommander": ["iOS"],
     "spoon": ["Android"],
-    "sqldelight": ["Android"],
+    "sqldelight": ["Android", "Kotlin"],
```

Comment:
@alecholmes Should we add this to iOS too, since it's MPP?

Reply:
yeeeeee

Reply:
Done.